relay-event-normalization/src/normalize/span/description/redis.rs

```diff
@@ -0,0 +1,598 @@
+    "SAVE ",
+    "SCAN ",
+    "SCARD ",
+    "SCRIPT ",
+    "SCRIPT DEBUG ",
+    "SCRIPT EXISTS ",
+    "SCRIPT FLUSH ",
+    "SCRIPT HELP ",
+    "SCRIPT KILL ",
+    "SCRIPT LOAD ",
+    "SDIFF ",
+    "SDIFFSTORE ",
+    "SELECT ",
+    "SET ",
+    "SETBIT ",
+    "SETEX ",
+    "SETNX ",
+    "SETRANGE ",
+    "SHUTDOWN ",
+    "SINTER ",
+    "SINTERCARD ",
+    "SINTERSTORE ",
+    "SISMEMBER ",
+    "SLAVEOF ",
+    "SLOWLOG ",
+    "SLOWLOG GET ",
+    "SLOWLOG HELP ",
+    "SLOWLOG LEN ",
+    "SLOWLOG RESET ",
+    "SMEMBERS ",
+    "SMISMEMBER ",
+    "SMOVE ",
+    "SORT ",
+    "SORT_RO ",
+    "SPOP ",
+    "SPUBLISH ",
+    "SRANDMEMBER ",
+    "SREM ",
+    "SSCAN ",
+    "SSUBSCRIBE ",
+    "STRLEN ",
+    "SUBSCRIBE ",
+    "SUBSTR ",
+    "SUNION ",
+    "SUNIONSTORE ",
+    "SUNSUBSCRIBE ",
+    "SWAPDB ",
+    "SYNC ",
+    "TDIGEST.ADD ",
+    "TDIGEST.BYRANK ",
+    "TDIGEST.BYREVRANK ",
+    "TDIGEST.CDF ",
+    "TDIGEST.CREATE ",
+    "TDIGEST.INFO ",
+    "TDIGEST.MAX ",
+    "TDIGEST.MERGE ",
+    "TDIGEST.MIN ",
+    "TDIGEST.QUANTILE ",
+    "TDIGEST.RANK ",
+    "TDIGEST.RESET ",
+    "TDIGEST.REVRANK ",
+    "TDIGEST.TRIMMED_MEAN ",
+    "TIME ",
+    "TOPK.ADD ",
+    "TOPK.COUNT ",
+    "TOPK.INCRBY ",
+    "TOPK.INFO ",
+    "TOPK.LIST ",
+    "TOPK.QUERY ",
+    "TOPK.RESERVE ",
+    "TOUCH ",
+    "TS.ADD ",
+    "TS.ALTER ",
+    "TS.CREATE ",
+    "TS.CREATERULE ",
+    "TS.DECRBY ",
+    "TS.DEL ",
+    "TS.DELETERULE ",
+    "TS.GET ",
+    "TS.INCRBY ",
+    "TS.INFO ",
+    "TS.MADD ",
+    "TS.MGET ",
+    "TS.MRANGE ",
+    "TS.MREVRANGE ",
+    "TS.QUERYINDEX ",
+    "TS.RANGE ",
+    "TS.REVRANGE ",
+    "TTL ",
+    "TYPE ",
+    "UNLINK ",
+    "UNSUBSCRIBE ",
+    "UNWATCH ",
+    "WAIT ",
+    "WAITAOF ",
+    "WATCH ",
+    "XACK ",
+    "XADD ",
+    "XAUTOCLAIM ",
+    "XCLAIM ",
+    "XDEL ",
+    "XGROUP ",
+    "XGROUP CREATE ",
+    "XGROUP CREATECONSUMER ",
+    "XGROUP DELCONSUMER ",
+    "XGROUP DESTROY ",
+    "XGROUP HELP ",
+    "XGROUP SETID ",
+    "XINFO ",
+    "XINFO CONSUMERS ",
+    "XINFO GROUPS ",
+    "XINFO HELP ",
+    "XINFO STREAM ",
+    "XLEN ",
+    "XPENDING ",
+    "XRANGE ",
+    "XREAD ",
+    "XREADGROUP ",
+    "XREVRANGE ",
+    "XSETID ",
+    "XTRIM ",
+    "ZADD ",
+    "ZCARD ",
+    "ZCOUNT ",
+    "ZDIFF ",
+    "ZDIFFSTORE ",
+    "ZINCRBY ",
+    "ZINTER ",
+    "ZINTERCARD ",
+    "ZINTERSTORE ",
+    "ZLEXCOUNT ",
+    "ZMPOP ",
+    "ZMSCORE ",
+    "ZPOPMAX ",
+    "ZPOPMIN ",
+    "ZRANDMEMBER ",
+    "ZRANGE ",
+    "ZRANGEBYLEX ",
+    "ZRANGEBYSCORE ",
+    "ZRANGESTORE ",
+    "ZRANK ",
+    "ZREM ",
+    "ZREMRANGEBYLEX ",
+    "ZREMRANGEBYRANK ",
+    "ZREMRANGEBYSCORE ",
+    "ZREVRANGE ",
+    "ZREVRANGEBYLEX ",
+    "ZREVRANGEBYSCORE ",
+    "ZREVRANK ",
+    "ZSCAN ",
+    "ZSCORE ",
+    "ZUNION ",
+    "ZUNIONSTORE ",
+];
+
+static REDIS_COMMAND_MATCHER: Lazy<AhoCorasick> = Lazy::new(|| {
+    AhoCorasickBuilder::new()
```

Comment:
I think you want to set this to an anchored search mode `start_kind(StartKind::Anchored)`?